reactioncommerce/kinetic
# Implement Breadcrumbs component

Resolves #92
Figma: https://www.figma.com/file/vWot2cXXgjcsLpMq4SAX90/Kinetic?node-id=1717%3A13491

Testing Instructions:
- Start the app and login with a valid account
- Breadcrumbs component should display on  all pages
- If breadcrumbs have more than 2 items, it will display '...' in the middle
- Click '...' to display all breadcrumbs items
 
Screenshots
<img width="2028" alt="Screen Shot 2022-10-31 at 10 14 46" src="https://user-images.githubusercontent.com/33818318/198923632-4421826f-9350-4af4-9fa9-fa132cd81b2e.png">
<img width="2028" alt="Screen Shot 2022-10-31 at 10 14 23" src="https://user-images.githubusercontent.com/33818318/198923638-11df6feb-7208-44b7-96aa-b424dbd189f5.png">
